items/weapons/grenades.json

```diff
@@ -1,5 +1,5 @@
 {
-	"1": {
+	"3": {
```

Comment:
are these numbers supposed to change (again)?

Reply:
yee